dmscripts/send_dos_opportunities_email.py

```diff
@@ -0,0 +1,143 @@
+from datetime import datetime, date, timedelta
+
+from requests.exceptions import RequestException
+
+from dmscripts.helpers.html_helpers import render_html
+from dmscripts.helpers import logging_helpers
+from dmscripts.helpers.logging_helpers import logging
+from dmutils.formats import DATETIME_FORMAT, DISPLAY_DATE_FORMAT
+
+import dmapiclient
+
+logger = logging_helpers.configure_logger({'dmapiclient': logging.INFO})
+
+
+def get_live_briefs_between_two_dates(data_api_client, lot_slug, start_date, end_date):
+    return [
+        brief for brief in data_api_client.find_briefs_iter(status="live", lot=lot_slug)
+        if datetime.strptime(brief['publishedAt'], DATETIME_FORMAT).date() >= start_date
+        and datetime.strptime(brief['publishedAt'], DATETIME_FORMAT).date() <= end_date
+    ]
+
+
+def get_campaign_data(lot_name, list_id):
+    return {
+        "type": "regular",
+        "recipients": {
+            "list_id": list_id,
+        },
+        "settings": {
+            "subject_line": "New opportunities for {0}: Digital Outcomes and Specialists 2".format(lot_name),
+            "title": "DOS Suppliers: {0} [{1}]".format(lot_name, datetime.now().strftime("%d %B")),
+            "from_name": "Digital Marketplace Team",
+            "reply_to": "do-not-reply@digitalmarketplace.service.gov.uk",
```

Comment:
Looks like this value is hard coded in the supplier fe :/
https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/config.py#L47-L47

Maybe we could have it in a config file in the dmutils/email repo. Might be worth making a tech debt ticket for that. I'm happy to consider it as slightly out of scope for this ticket.